tests/test_mock_example.py

```
@@ -0,0 +1,51 @@
+"""Example tests demonstrating mock_gh_command fixture usage."""
+
+import subprocess
+
+
+def test_mock_gh_command_default_behavior(mock_gh_command):
+    """Test that mock_gh_command provides realistic default behavior."""
+    # Test default successful behavior
+    result = subprocess.run(["gh", "api", "user"], capture_output=True, text=True)
```

Pylint (3.3.7)

[warning] 'subprocess.run' used without explicitly defining the value for 'check'.

(W1510)